packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingsView.tsx

```diff
@@ -114,13 +114,13 @@ export const MappingsView = ({
           </div>
           <div className="gmw-button-spacing">
             <ExtractionStatusIcon
-              iconStatus={extractionIconData.iconStatus}
+              iconStatus={extractionStatusData.iconStatus}
```

Comment:
What happens when the `iconStatus` is undefined?

Reply:
`iconStatus` is initialized in context so whenever it's blue it appears in color blue.